### PR TITLE
fix(client): use sans-serif font for titles instead of monospace

### LINKED ISSUE #54047: Removing Monospace from titles
### Describe the Issue

Given that sans-serif fonts are generally more legible and more readable than monospace fonts, I recommend transitioning any remaining elements to our sans-serif font family (--font-family-sans-serif).

While font pairing is a recognized strategy for enhancing brand identity and adapting to different contexts, I believe we can achieve visual differentiation by varying only the size and weight of the Lato font. 

For simplicity, I propose using the --font-family-monospace exclusively for code blocks. For all other text, we should use Lato.

### PR DESCRIPTION
**Pull request by ajcruise007**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Uses sans-serif font for titles instead of monospace.

Closes #54047

### CODE REVIEW

File: client/src/components/layouts/global.css [hunk 1]
```diff
@@ -66,28 +66,28 @@ h1 {
   color: var(--secondary-color);
   font-weight: 700;
   font-size: 1.5rem;
-  font-family: var(--font-family-monospace);
+  font-family: var()--font-family-sans-serif;
```
Comment (naomi-lgbt) [inline client/src/components/layouts/global.css:69]:
```suggestion
  font-family: var(--font-family-sans-serif);
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: client/src/components/layouts/global.css (hunk 1 above)
Comment (ajcruise007) [reply client/src/components/layouts/global.css]: Changes done @naomi-lgbt , was a typo my bad.

**Comment by ajcruise007** [comment]
hi @huyenltnguyen , how will I come to know that the issue is open for contribution?

**Comment by ajcruise007** [comment]
I'm a little bit new to open source, but I would love to contribute to this repo.

**Comment by ahmaxed** [comment]
@ajcruise007,  let me know if you have any questions.

**Review by huyenltnguyen — APPROVED** [review]
I've updated the PR to remove the font family overrides.

They are no longer needed as sans-serif is the default font family:

https://github.com/freeCodeCamp/freeCodeCamp/blob/b0d547e07addea7dac45471b60c5ca2154f0d484/client/src/components/layouts/global.css#L17-L19

**Review by Sembauke — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**